app/init.php

```diff
@@ -847,9 +848,12 @@ function (mixed $value) {
 App::setResource('deletes', fn() => new Delete());
 App::setResource('database', fn() => new EventDatabase());
 App::setResource('messaging', fn() => new Phone());
-App::setResource('queueForFunctions', function (Group $pools) {
-    return new Func($pools->get('queue')->pop()->getResource());
+App::setResource('queueConnection', function (Group $pools) {
```

Comment:
why `queueConnection`? is `queue` already being used somewhere?

Reply:
Renamed to queue